### DISCUSSION TITLE
用python進行股票分析

### DISCUSSION THREAD

**Discussion by wu51000** [1101-GS4719-C/Python-Community-GS4719-C#990] [優秀Python套件]
憑著小資族要學會投資理財的精神，查找了如何用python分析股市股價的網站。
沒想到真的有很多很厲害的分析工具、模組…..長知識了…..
當然雞蛋不能放同個籃子裡，但若有好工具的輔助可以幫助我們事半功倍！
[https://pyradise.com/用-python-與-excel-打造簡易的回測系統-857bfbb88867](url)
https://pyradise.com/用-python-與-excel-打造簡易的回測系統-857bfbb88867
（連結不知為何點不進去……我補個新在這！）

**Comment by ruthchen1020** [comment]
連結好像不能用喔...
但是用python分析股票好像很多人有在做了。

**Reply by wu51000** [reply]
> 連結好像不能用喔...
> 但是用python分析股票好像很多人有在做了。

有放新的了可以去看看！
雖然很多人在做了 但我想可能還是有不知到這個資訊的同學！

**Comment by peggy0329** [comment]
雖然很多人有做過這個應用～
但直接分享資訊給本科生！我覺得很有用！感謝分享

**Comment by manipulateed** [comment]
其實這篇主要是透過xlwings的使用去將複雜的功能簡化，再套用到股票的回測上面，股票一直都是飄忽不定的東西，投資不是只有數據，數據也無法及時反應真實的市場。

**Comment by harrylee1971** [comment]
之後有空來研究研究這個，這樣子或許我的股票顏色就是紅通通不是綠油油了QAQ

**Comment by a45320614** [comment]
股票的歷史資料其實也可以透過爬蟲的方式來獲取，然後再將爬下來的資料與文章中提到的xlwings這個套件進行結合運用，製造一個可以達到全自動化的股價回測系統，也可以再結合資料科學相關的工具進行資料分析，模擬真實的交易狀況，並且驗證自己的投資策略。
